data/Services.json

```diff
@@ -4448,14 +4448,21 @@
                 "ref": "https://studio.azureml.net/",
                 "icon": "Azure Machine Learning.png"
             },
+
             {
-                "name": "Azure Machine Learning Workbench",
-                "ref": "https://docs.microsoft.com/en-in/azure/machine-learning/preview/quickstart-installation",
+                "name": "Azure Cognitive Services",
```

Comment:
I just realized there is an AI section so please cancel this commit. I will commit a new change